discovery/googleapis/firestore__v1.json

```diff
@@ -3088,23 +3088,26 @@
         "RunAggregationQueryResponse": {
             "description": "The response for Firestore.RunAggregationQuery.",
             "id": "RunAggregationQueryResponse",
-            "properties": {
-                "readTime": {
-                    "description": "The time at which the aggregate value is valid for.",
-                    "format": "google-datetime",
-                    "type": "string"
-                },
-                "result": {
-                    "$ref": "AggregationResult",
-                    "description": "A single aggregation result. Not present when reporting partial progress."
+            "items": {
```

Comment:
Need to add a `diff` file here. But kudos for figuring this out!

Reply:
Thanks!  I wasn't sure this would be the right way to change the discovery docs.

I think I see what `generator.dart` wants (_'Applying diffs!_'), I'll give that a shot.